ci/scripts/github/test.sh

```diff
@@ -19,7 +19,12 @@ set -e
 source ${WORKSPACE}/ci/scripts/github/common.sh
 /usr/bin/nvidia-smi
 
-update_conda_env
+rapids-dependency-file-generator \
+  --output conda \
+  --file_key test \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch);py=${RAPIDS_PY_VERSION}" | tee env.yaml
```

Comment:
Should this just happen in the `update_conda_env` function?

Reply:
No I don't think so. Every other RAPIDS scripts does it at this level.